WebService/v1/Models/DevicePropertiesApiModel.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+// TODO: tests
+// TODO: handle errors
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models
+{
+    public class DevicePropertiesApiModel
+    {
+        [JsonProperty(PropertyName = "Reported")]
+        public HashSet<string> ReportedProperties { get; set; }
```

Comment:
When returning a list we should follow the convention in place for lists, e.g. see `DeviceModelListApiModel.cs`. The response should be something like:
```
{
    "Items": [
       {
           "Id": "..."
       },
       {
           "Id": "..."
       },
       {
           "Id": "..."
       },
    ]
}
```

Reply:
done